docs/api/v1.md

```diff
@@ -106,12 +292,89 @@ to stream trace with chunked encoding, maybe omit status).
 }
 ```
 
+
 ### Execution
 ```
 {
-    id {int}: Unique Identifier
-    otherInfo {object}: TODO Whatever other info, ask Russell 
+   id {integer}: Unique Identifier
+   type {string}: Type of execution. Either `function` or `process`
+   status {string}: Either `pending`, `executing`, `stopped`, or `done`
+   executionTime {integer}: Nanoseconds
+   data {object}: Data specific to this type
+      if `type=function`: {
+         function {[Function](#function)}: Function that produced this execution
+      }
+      if `type=process`: {
+         nextExecution {Execution id}: Id of the execution that follows this one. Null until `status=done`. Initially set to `null`, set when a call to `/debug/execute` hits a breakpoint, in which case it points to the breakpoint function�s execution.
+      }
 }
 ```
 
+
+### Trace
+```
+{
+   index {integer}: Index of the trace, beginning at 0 and totally ordered for each execution
+   type {integer}: Type of trace: `0`=instruction (state changes), `1`=output (to stdout), `2`=termination (data about how the execution ended)
```

Comment:
Is this enough information to generate the visualizations we have? Or would it help to add a control flow trace for branches/loops/calls?